pages/index.js

```diff
@@ -1,102 +1,174 @@
+import { makeStyles } from "@material-ui/core/styles";
 
-import schema from '../schemas/schema';
+import { Octokit } from "@octokit/core";
+import { createPullRequest } from "octokit-plugin-create-pull-request";
 
-const validatorMapper = {}
+import schema from "../schemas/schema";
+
+const validatorMapper = {};
+
+const GITHUB_OWNER = "nathanbaleeta";
+const GITHUB_REPO = "submission-form";
+const GITHUB_BRANCH = "main"; /* optional: defaults to default branch */
+
+const TOKEN = process.env.ACCESS_TOKEN; // create token at https://github.com/settings/tokens/new?scopes=repo
 
 const useStyles = makeStyles((theme) => ({
   paper: {
     marginTop: theme.spacing(8),
-    display: 'flex',
-    flexDirection: 'column',
-    alignItems: 'center',
+    display: "flex",
+    flexDirection: "column",
+    alignItems: "center",
   },
   avatar: {
     margin: theme.spacing(1),
     backgroundColor: theme.palette.secondary.main,
   },
   form: {
-    width: '100%', // Fix IE 11 issue.
+    width: "100%", // Fix IE 11 issue.
     marginTop: theme.spacing(1),
   },
   submit: {
     margin: theme.spacing(3, 0, 2),
-  }
-
+  },
 }));
 
 const theme = createMuiTheme({
   overrides: {
     MuiButton: {
       root: {
-        background: 'linear-gradient(45deg, #FE6B8B 30%, #FF8E53 90%)',
+        background: "linear-gradient(45deg, #FE6B8B 30%, #FF8E53 90%)",
         borderRadius: 3,
         border: 0,
-        color: 'white',
+        color: "white",
         height: 48,
-        padding: '0 30px',
-        boxShadow: '0 3px 5px 2px rgba(255, 105, 135, .3)',
+        padding: "0 30px",
+        boxShadow: "0 3px 5px 2px rgba(255, 105, 135, .3)",
       },
     },
     MuiCheckbox: {
       root: {
-        padding: '1px 9px'
-      }
-    }
+        padding: "1px 9px",
+      },
+    },
   },
 });
 
-const MyFormTemplate = (props) => <FormTemplate {...props} showFormControls={false} />
+const MyFormTemplate = (props) => (
+  <FormTemplate {...props} showFormControls={false} />
+);
 
 const validate = (values) => {
-  console.log(values)
+  console.log(values);
   const errors = {};
 
-  for(let i=1; i<=17; i++){
-    if (values.SDGs && values.SDGs.includes(i) && ! values['evidenceText'+i] && ! values['evidenceURL'+i]) {
-      errors['evidenceText'+i] = 'Either the description or a URL is required';
-      errors['evidenceURL'+i] = 'Either the description or a URL is required';
+  for (let i = 1; i <= 17; i++) {
+    if (
+      values.SDGs &&
+      values.SDGs.includes(i) &&
+      !values["evidenceText" + i] &&
+      !values["evidenceURL" + i]
+    ) {
+      errors["evidenceText" + i] =
+        "Either the description or a URL is required";
+      errors["evidenceURL" + i] = "Either the description or a URL is required";
     }
   }
 
   return errors;
+};
+
+function openPR(values) {
+  let sdgNumber, evidenceText;
+  for (let i = 0; i < values.SDGs.length; i++) {
+    sdgNumber = JSON.stringify(values.SDGs[i]);
```

Comment:
Why is this needed? couldn't you combine L97 and L98 into one as follows?
```
sdgNumber = parseInt(values.SDGs[i])
```

Reply:
Resolved in commit 8988263.